packages/kafka-iam-auth/src/index.ts

```diff
@@ -242,7 +242,12 @@ const initCustomConsumer = async ({
   config: KafkaConsumerConfig;
   topics: string[];
   consumerRunConfig: (consumer: Consumer) => ConsumerRunConfig;
-  batchConfig?: { minBytes: number; maxWaitTimeInMs: number };
+  batchConfig?: {
+    minBytes: number;
+    maxWaitTimeInMs: number;
+    sessionTimeout: number;
+    maxBytes: number | undefined;
```

Comment:
adding `maxBytes` to the configs in the commons is a good idea, let's move the [calculation](https://github.com/pagopa/interop-be-monorepo/pull/1308/files#diff-ca1d61904ab1093f1b5705746d25c9f89fa701af27671f5d97d7cba3039f15e0R421) inside the consumer and ere just use the given value